lego/apps/users/notifications.py

```diff
@@ -30,3 +36,47 @@ def generate_push(self):
             context={"weight": penalty.weight, "event": penalty.source_event.title},
             instance=penalty,
         )
+
+
+class InactiveNotification(Notification):
+
+    name = INACTIVE_WARNING
+
+    def generate_mail(self):
+        max_inactive_days = self.kwargs["max_inactive_days"]
+
+        return self._delay_mail(
+            to_email=self.user.email,
+            context={
+                "name": self.user.first_name,
+                "username": self.user.username,
+                "last_login": str(self.user.last_login.date()),
+                "date_of_deletion": str(
+                    (self.user.last_login + timedelta(max_inactive_days)).date()
+                ),
+                "max_inactive_days": max_inactive_days,
```

Comment:
I feel like this should say something more serious, like "Your account will soon be deleted" (or "Din konto på abakus.no vil snart slettes").

Otherwise this could easily be dismissed without reading the email.

Reply:
Very much agree!